src/sidebar/test/oauth-auth-test.js

```diff
@@ -53,6 +53,18 @@ describe('oauth auth', function () {
       });
     });
 
+    it('should raise if a grant token request fails', function () {
```

Comment:
Hmm, looks like the spec calls them "access token requests" and "refresh requests" :)